libbpfgo/libbpfgo.go

```diff
@@ -23,6 +23,14 @@ extern void perfLostCallback(void *ctx, int cpu, __u64 cnt);
 
 extern int ringbufferCallback(void *ctx, void *data, size_t size);
 
+#define MAX_ERRNO       4095
```

Comment:
Can you please add a comment here that this was taken from err.h of libbpf?

Reply:
Yes https://github.com/aquasecurity/tracee/pull/694/commits/1f4717ba14be975d4efe2aee3ee9f84b25aade57#diff-4108726d54769f59926ec61abdeaab04a651cdf8766a24a74a859b4d98befbc8R37